test/intl402/DurationFormat/prototype/format/basic-format-en.js

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.DurationFormat.prototype.format
+description: Test if format method formats duration correctly
+locale: [en-US]
+features: [Intl.DurationFormat]
+---*/
+
+const duration = {
+  years: 1,
+  months: 2,
+  weeks: 3,
+  days: 3,
+  hours: 4,
+  minutes: 5,
+  seconds: 6,
+  milliseconds: 7,
+  microseconds: 8,
+  nanoseconds: 9,
+};
+
+const df = new Intl.DurationFormat('pt');
```

Comment:
```suggestion
const df = new Intl.DurationFormat('en');
```
The name of the test and the expected value indicate English. (Or this could be `en-US` as the test frontmatter indicates?)